From 18ae4e4a8d2c33f2b1e6b4187c2d1aa6bcc51d0f Mon Sep 17 00:00:00 2001
From: Muhammad Alfi Syahrin <malfisya.dev@hotmail.com>
Date: Wed, 21 Feb 2024 14:17:15 +0700
Subject: [PATCH] Fix desktop files

Stop using absolute path for exec and icon in desktop files
---
 hp-uiscan.desktop.in     | 4 ++--
 hplip-systray.desktop.in | 2 +-
 hplip.desktop.in         | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hp-uiscan.desktop.in b/hp-uiscan.desktop.in
index 4a551e5..42c2388 100644
--- a/hp-uiscan.desktop.in
+++ b/hp-uiscan.desktop.in
@@ -4,5 +4,5 @@ Version=1.0
 Type=Application
 Terminal=false
 Name=hp-uiscan
-Exec=/usr/bin/hp-uiscan
-Icon=/usr/share/icons/Humanity/devices/48/printer.svg
+Exec=hp-uiscan
+Icon=hp-logo
diff --git a/hplip-systray.desktop.in b/hplip-systray.desktop.in
index 870dec6..85364e2 100644
--- a/hplip-systray.desktop.in
+++ b/hplip-systray.desktop.in
@@ -5,7 +5,7 @@ Name=HP System Tray Service
 GenericName=Printer Status Applet
 Comment=HP System Tray Service
 Exec=hp-systray -x
-Icon=@abs_datadir@/hplip/data/images/128x128/hp_logo.png
+Icon=hp-logo
 Terminal=false
 Categories=Application;Utility;
 X-KDE-StartupNotify=false
diff --git a/hplip.desktop.in b/hplip.desktop.in
index 5a57f5b..809a391 100644
--- a/hplip.desktop.in
+++ b/hplip.desktop.in
@@ -5,7 +5,7 @@ Name=HP Device Manager
 GenericName=Printer Management Application
 Comment=View device status, ink levels and perform maintenance.
 Exec=hp-toolbox
-Icon=@abs_datadir@/hplip/data/images/128x128/hp_logo.png
+Icon=hp-logo
 Terminal=false
 Categories=Application;Utility;
 X-KDE-StartupNotify=false
-- 
2.43.1

